website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,133 @@
+
+- Enabling you to materialize these queries within your data platform using dbt Cloud.
+- Proving an integration path for tools that don't natively support the dbt Semantic Layer by exposing tables of metrics and dimensions.
+
+Essentially, Exports are like any other table in your data platform. They allow you to query metric definition through any SQL interface or connect to downstream tools without needing a first class Semantic Layer integration.
+
+## Prerequisites
+
+- You have a dbt Cloud account on a [Team or Enterprise](https://www.getdbt.com/pricing/) plan.
+- You are on [dbt version](/docs/dbt-versions/upgrade-core-in-cloud) 1.7 or higher.
+- You have the dbt Semantic Layer [configured](/docs/use-dbt-semantic-layer/setup-sl) in your dbt project.
+- You have a dbt Cloud environment with a [Job scheduler](/docs/deploy/job-scheduler) enabled.
+
+### Exports and Saved Queries comparison
+
+The following table compares the features and usage between Exports and Saved Queries: 
+
+| Feature |  Exports | <div style={{width:'250px, text-align: center'}}>Saved Queries</div>  |
+| ----------- | ----------- | ---------------- |
+| **Availability**    | Available on dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) plans (dbt versions 1.7 or higher).| Available in both dbt Core and dbt Cloud.     |
+| **Purpose**         | To materialize Saved Queries in your data platform and expose metrics and dimensions as a view or table. | To define and manage common Semantic Layer queries in YAML, which includes metrics and dimensions.   |
+| **Usage**           | Automatically runs Saved Queries and materializes them within your data platform. Exports count towards [queried metrics](/docs/cloud/billing#what-counts-as-a-queried-metric) usage. <br /><br />**Example**: Creating a weekly aggregated table for active user metrics, automatically updated and stored in the data platform.  | Used for organizing and reusing common MetricFlow queries within dbt projects.<br /><br /><br />**Example**: Group related metrics together for better organization, and include commonly uses dimensions and filters. | For materializing query results in the data platform. |
+| **Integration**     | Must have the dbt Semantic Layer configured in your dbt project.<br /><br />Tightly integrated with the [MetricFlow Server](/docs/use-dbt-semantic-layer/sl-architecture#components) and dbt Cloud's Job scheduler. | Integrated into the dbt <Term id="dag" /> and managed alongside other dbt nodes. |
+| **Configuration**   | Configured within dbt Cloud environment and Job scheduler settings. | Defined in YAML format within dbt project files.   |
+
+## Define Exports
+
+Exports are an additional configuration added to a Saved Query. They define how to materialize a Saved Query, along with the schema and table name. 
+
+You can define Exports in a YAML format as a key within the `saved_queries` configuration. Here's an example of a Saved Query with an Export:
+
+```yaml
```

Comment:
@DevonFulcher do we add saved queries and exports in their own 'saved_queries.yml' file?

Reply:
That is what I have been doing. I'm not sure what the best practice is. Maybe it should be in the same file as the semantic models? I'm not sure @Jstein77 would know best.

Reply:
Devon's spot on, I would add them in the same file as my metric definitions.